shared-libs/cht-datasource/src/person.ts

```diff
@@ -110,4 +139,18 @@ export namespace v1 {
    * @throws Error if a data context is not provided
    */
   export const getPage = getPeople(Local.Person.v1.getPage, Remote.Person.v1.getPage);
+
+  /**
+   * Returns a people generator function for fetching batches of people data.
+   * @param context the current data context
+   * @returns a generator function that creates an iterator-like object for fetching people data.
+   * @example
+   * const getAllIterator = Person.v1.getAll(ctx)(Qualifier.byContactType('person'));
+   * for await (const page of getAllIterator) {
+   *   for (const doc of page) {
+   *     ...
+   *   }
+   * }
```

Comment:
Based on my earlier suggestion, I think the generator should yield individual Person objects (and not batches).  Without the batches, consuming this function should be much more straightforward and I do not think we need an example.

```suggestion
   * Returns a function for retrieving people from the given data context.
   * @param context the current data context
   * @returns a function for retrieving people
   * @throws Error if a data context is not provided
```